src/django/api/views/facility/facility_list_view_set.py

```diff
@@ -390,103 +385,16 @@ def createlist(self, request):
-                    lng = float(row.fields[FileHeaderField.LNG])
-                    item.geocoded_point = Point(lng, lat)
-                    is_geocoded = True
-
-                create_extendedfields_for_listitem(
-                    item,
-                    list(row.fields.keys()),
-                    list(row.fields.values())
-                )
-            except Exception as e:
-                log.error(
-                    f'[List Upload] Creation of ExtendedField error: {e}'
-                )
-                log.info(f'[List Upload] FacilityListItem Id: {item.id}')
-                item.status = FacilityListItem.ERROR_PARSING
-                item.processing_results.append({
-                    'action': ProcessingAction.PARSE,
-                    'started_at': parsing_started,
-                    'error': True,
-                    'message': str(e),
-                    'trace': traceback.format_exc(),
-                    'finished_at': str(timezone.now()),
-                    'is_geocoded': is_geocoded,
-                })
-
-            row_errors = row.errors
-            if len(row_errors) > 0:
-                stringified_message = '\n'.join(
-                    [f"{error['message']}" for error in row_errors]
-                )
-                log.error(
-                    f'[List Upload] CC Parsing Error: {stringified_message}'
-                )
-                log.info(f'[List Upload] FacilityListItem Id: {item.id}')
-                item.status = FacilityListItem.ERROR_PARSING
-                item.processing_results.append({
-                    'action': ProcessingAction.PARSE,
-                    'started_at': parsing_started,
-                    'error': True,
-                    'message': stringified_message,
-                    'trace': traceback.format_exc(),
-                    'finished_at': str(timezone.now()),
-                    'is_geocoded': is_geocoded,
-                })
-            else:
-                item.status = FacilityListItem.PARSED
-                item.processing_results.append({
-                    'action': ProcessingAction.PARSE,
-                    'started_at': parsing_started,
-                    'error': False,
-                    'finished_at': str(timezone.now()),
-                    'is_geocoded': is_geocoded,
-                })
-
-            if item.status != FacilityListItem.ERROR_PARSING:
-                core_fields = '{}-{}-{}'.format(item.country_code,
-                                                item.clean_name,
-                                                item.clean_address)
-                if core_fields in parsed_items:
-                    item.status = FacilityListItem.DUPLICATE
-                else:
-                    parsed_items.add(core_fields)
-
-            item.save()
-
         serializer = self.get_serializer(new_list)
-        return Response(serializer.data)
+
+        return ProcessingFacility.createList(
```

Comment:
"create_list" python style method name.
Also please fetch latest changes and pass processed_data instead of rows to the method, prepare the similar as it implemented for API upload on latest changes.

Reply:
Thank you for your comment. Here are the changes that were made:
- method name `createList` was changed to `create_list`
- `rows` are initialized at line 359 ` rows = processed_data.rows`
- branch 'сс-fix-facility-uploads' was merged into the current branch